pyproject.toml

```diff
@@ -18,7 +18,7 @@ dependencies = [
     "opentelemetry-instrumentation-requests>=0.54b1",
     "opentelemetry-instrumentation-starlette>=0.54b0",
     "asgi-lifespan>=2.1.0",
-    "slim-bindings==0.4.1",
+    "slim-bindings==0.5.0",
```

Comment:
Do we have to use 0.5.0? If not, can we keep it at v0.4.x since the support for 0.6.0 is already on the way